dftimewolf/lib/exporters/local_filesystem.py

```diff
@@ -28,32 +29,46 @@ def SetUp(self, target_directory=None):  # pylint: disable=arguments-differ
       target_directory (Optional[str]): path of the directory in which
           collected files will be copied.
     """
-    self._target_directory = target_directory
     if not target_directory:
-      self._target_directory = tempfile.mkdtemp()
-    elif not os.path.exists(target_directory):
-      os.makedirs(target_directory)
+      target_directory = tempfile.mkdtemp()
```

Comment:
Please add a prefix or suffix here so the directory can be recognized later

Reply:
Done.